demisto_sdk/commands/common/tools.py

```diff
@@ -812,6 +813,8 @@ def get_file(
 
     try:
         file_content = _read_file(file_path)
+        if return_content:
+            return file_content
```

Comment:
```suggestion
        if return_content := _read_file(file_path):
            return file_content
```

Reply:
This is not the correct behavior.
We want to read the file anyway, and return its content only of return_content is True

Reply:
You're right, it's getting late..